hyp3_gamma/__main__.py

```diff
@@ -58,10 +59,30 @@ def check_earthdata_credentials(username, password):
     return username, password
 
 
+def check_esa_credentials(username: Optional[str], password: Optional[str]) -> None:
```

Comment:
My preference would be to see both this function and the `check_earthdata_credentials` in utils.py, but it's not blocking.